xarray/backends/h5netcdf_.py

```diff
@@ -431,11 +430,72 @@ def open_dataset(  # type: ignore[override]  # allow LSP violation, not supporti
     def open_datatree(
         self,
         filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+        *,
+        mask_and_scale=True,
+        decode_times=True,
+        concat_characters=True,
+        decode_coords=True,
+        drop_variables: str | Iterable[str] | None = None,
+        use_cftime=None,
+        decode_timedelta=None,
+        format=None,
```

Comment:
```suggestion
	mode="r",
        format=None,
```
I am still trying to figure out how you gathered these keyword args, but the h5netcdf store's open, takes a mode.

Reply:
@aladinor This was this only thing remaining for me.  And we're talking about it now.  I'm fine with the separate backends. Looks good.